### ISSUE TITLE
Fix Broken Absolut Imports, Documentation, Tests in GANs #1

### ISSUE THREAD

**Issue by someshsingh22** [Adversarial-Deep-Learning/code-soup#16]
- [ ] Absolute imports are broken, relative / fix them
- [ ] DocStrings, References are missing
- [ ] Fix Coverage Tests

**Comment by Devanshu24** [comment]
Regarding the first point, I think the imports are working right?
Any specific that you found to be broken?

**Comment by someshsingh22** [comment]
The imports are working because the imports in mnist_gan.py are direct, the __init__.py has some imports that are not working and the build cases pass, I will put a detailed error log in a while

**Comment by someshsingh22** [comment]
scripts written in code_soup or subfolders would not run because the reference to modules is absolute, however, the tutorials were supposed to be following code soup structure which will need relative imports.

However according to PEP8: [Relative vs Absolute Imports](https://www.python.org/dev/peps/pep-0008/#:~:text=Absolute%20imports%20are%20recommended%2C%20as%20they%20are%20usually%20more%20readable%20and%20tend%20to%20be%20better%20behaved%20(or%20at%20least%20give%20better%20error%20messages)%20if%20the%20import%20system%20is%20incorrectly%20configured%20(such%20as%20when%20a%20directory%20inside%20a%20package%20ends%20up%20on%20sys.path)%3A)
Relative Imports are discouraged, so we will follow the layout for tutorials outside the main code.
Changing the tasks and issue.

**Comment by abhi-glitchhg** [comment]
Can I take up this task?

**Comment by someshsingh22** [comment]
Yes @abhi-glitchhg you can take this up, please keep updating your progress/plan
